src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../Tab';
+
+interface Props {
+  tabs: Tab[];
+  setSelectedTab: (tab: Tab) => void,
+  selectedTab: Tab,
+}
+
+export const Tabs: FC<Props> = ({ tabs, setSelectedTab, selectedTab }) => {
+  const handleClick = (tab: Tab) => {
+    setSelectedTab(tab);
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab: Tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames({
+                  'is-active': selectedTab.id === id,
```

Comment:
move it to the variable
```suggestion
                  'is-active': ...,
```